include/mgos_pcf857x.h

```diff
@@ -63,6 +66,14 @@ void mgos_pcf857x_gpio_clear_int(struct mgos_pcf857x *dev, int pin);
 void mgos_pcf857x_gpio_remove_int_handler(struct mgos_pcf857x *dev, int pin, mgos_gpio_int_handler_f *old_cb, void **old_arg);
 bool mgos_pcf857x_gpio_set_button_handler(struct mgos_pcf857x *dev, int pin, enum mgos_gpio_pull_type pull_type, enum mgos_gpio_int_mode int_mode, int debounce_ms, mgos_gpio_int_handler_f cb, void *arg);
 
+/*
+ * A utility function that takes care of blinking an LED.
+ * The pin must be configured as output first.
+ * Set to (0, 0) to disable.
+ */
+bool mgos_pcf857x_gpio_blink(struct mgos_pcf857x *dev, int pin, int on_ms, int off_ms);
+void mgos_pcf857x_gpio_blink_cb(void *arg);
```

Comment:
this is not a publicly usable function proto. Please move to src/*_internal.h

Reply:
Agreed. It has been moved.